unlock-app/src/hooks/useMembers.js

```diff
@@ -131,12 +135,17 @@ export const useMembers = (lockAddresses, viewer, filter) => {
     setLoading(true)
     const membersForLocksPromise = keyHolders.locks.map(async lock => {
       // If the viewer is not the lock owner, just show the members from chain
-      if (lock.owner.toLowerCase() !== viewer.toLowerCase()) {
+      const isLockManager = await web3Service.isLockManager(
```

Comment:
I am a bit worried about "flashes" here since there is going to be a small latency to check `isLockManager`. This means we'll start rendering the "not a lock manager" branch and then will render the "a lock manager" branch.

Since the default value is false, I also wonder if ti will not re-render twice even in the case of the viewer not being the lock mamager?

Maybe it would be good to add a `const [loading, setLoading] = useState(true)` to the hook and return early while rendering?